corehq/apps/userreports/es/data_source.py

```diff
@@ -113,24 +142,26 @@ def _get_aggregated_query(self, start, limit):
         for filter in self.filters:
             query = query.filter(filter)
 
-        top_agg = TermsAggregation(self.aggregation_columns[0], self.aggregation_columns[0], size=max_size)
-        for agg_column in self.aggregation_columns[1:]:
-            # todo support multiple aggregations
-            pass
+        innermost_agg_col = self.aggregation_columns[-1]
+        innermost_agg = TermsAggregation(innermost_agg_col, innermost_agg_col)
 
         aggregations = []
         for col in self.top_level_columns:
-            aggregations += col.aggregations(self.config, self.lang)
+            for agg in col.aggregations(self.config, self.lang):
+                innermost_agg.aggregation(agg)
+
+        top_agg = innermost_agg
+        for agg_column in self.aggregation_columns[:-1][::-1]:
```

Comment:
https://github.com/dimagi/commcare-hq/pull/14076/commits/04f1714996619ce6e0bafb3f324476d953d04fa7

not greatest comment, but hoping to do this in the helper in the future so it isn't so awful